src/common/utils.ts

```diff
@@ -123,7 +123,7 @@ export function isDescendant(parent: string, descendant: string): boolean {
 		parent += sep;
 	}
 
-	// Windows is case insensitive
+	// Windows is case-insensitive
```

Comment:
beyond the grammar this isn't technically correct. It depends on the file system and the configuration of the file system (or even folder? https://www.windowscentral.com/how-enable-ntfs-treat-folders-case-sensitive-windows-10 )